tls/s2n_resume.h

```diff
@@ -99,3 +99,11 @@ extern int s2n_store_to_cache(struct s2n_connection *conn);
  * once we release the session resumption API. */
 int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_t num);
 int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t num);
+
+typedef int (*s2n_session_ticket_cb)(struct s2n_connection *conn,
+                                     uint8_t *session_id_data,
+                                     size_t session_id_len,
+                                     uint8_t *session_data,
+                                     size_t session_len,
+                                     uint32_t session_lifetime);
```

Comment:
Nit: I like to keep the data and len parts of an argument together / on the same line when I have to wrap arguments. Might make this argument list more readable (same in your test implementation).